templates/.snapshots/TestRenderDeploymentDockerfile-deployments-appname-Dockerfile.tpl-deployments-testing-Dockerfile.snapshot

```diff
@@ -1,5 +1,5 @@
 (*codegen.File)(# syntax=docker/dockerfile:1.0-experimental
-FROM registry.example.com/foo/golang:1.23.4 as builder
+FROM registry.example.com/foo/golang:1.0.0 as builder
```

Comment:
As it turned out, the test was using the wrong config value and so it was grabbing the real golang version instead of the one defined in the test.